### PR TITLE
Pool detail page bottom section swap and liquidity

### PR DESCRIPTION
**Pull request by Huzaifa-Asif**

### CODE REVIEW

File: src/pages/explorePool.tsx [hunk 1]
```diff
@@ -48,46 +48,48 @@ export default function ExplorePool() {
...
+                </tr>
+                </thead>
+                <tbody>
+                    {
+                        tableData.map((item, index) => (
+                            <tr>
+                            <th>{index+1}</th>
+                            <td>
+                                <span> <a href={item.poolUrl} target='_blank' rel="noreferrer"> {item.poolAddress.substring(0,6)}...{item.poolAddress.substring(38,42)} <i className='fa fa-external-link text-dark' aria-hidden="true"></i> </a> </span>
+                                <br />
```
Comment (yawarshah1) [inline src/pages/explorePool.tsx:71]: please done use <br />

File: src/pages/explorePool.tsx [hunk 2]
```diff
@@ -48,46 +48,48 @@ export default function ExplorePool() {
...
+                                <span className="badge bg-warning text-dark">{item.poolType}</span>
+                            </td>
+                            <td>{item.feeRate}</td>
+                            <td className="chartColumn">
+                                <PieChart chartData={poolChartDataList} chartId={index} />
+                            </td>
+                            <td> 
+                                <span> <img src={duckIcon} alt=''></img> {item.liquidityValue1} {item.liquidityName1}</span> 
+                                <br />
```
Comment (yawarshah1) [inline src/pages/explorePool.tsx:80]: same here

**Review by yawarshah1 — CHANGES_REQUESTED** [review]

File: src/pages/explorePool.tsx (hunk 2 above)
Comment (Huzaifa-Asif) [reply src/pages/explorePool.tsx]: Resolved

File: src/pages/explorePool.tsx (hunk 1 above)
Comment (Huzaifa-Asif) [reply src/pages/explorePool.tsx]: Resolved

**Pull request merged**